libexec/bootstrap-scripts/main-deffile.sh

```diff
@@ -53,7 +53,7 @@ if [ -n "${BOOTSTRAP:-}" -a -z "${SINGULARITY_BUILDNOBASE:-}" ]; then
 fi
 
 # take a snapshot of the environment for later comparison
-if [ ${BOOTSTRAP:-} = "localimage" ]; then
+if [ "x${BOOTSTRAP:-}" = "xlocalimage" ]; then
```

Comment:
nitpick here,... if the variable is quoted, there is no need to append a character ('x') to the strings.